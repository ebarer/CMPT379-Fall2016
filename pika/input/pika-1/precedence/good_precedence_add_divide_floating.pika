exec {

  print 7.8 + 6.1 / 3.2, _n_.
  print 4.0 + 3.2 / 9.9, _n_.
  print 2.5 + 8.1 / 9.7, _n_.
  print 6.5 + 6.4 / 1.6, _n_.
  print 4.9 + 5.1 / 8.6, _n_.
  print 7.1 + 8.4 / 3.7, _n_.
  print 1.9 + 7.6 / 1.9, _n_.
  print 4.3 + 8.6 / 1.7, _n_.
  print 3.1 + 1.7 / 7.3, _n_.
  print 7.4 + 3.7 / 4.8, _n_.
  print 6.4 + 8.2 / 6.8, _n_.
  print 8.5 + 3.0 / 5.8, _n_.

  print 6.6 / 9.6 + 4.7, _n_.
  print 7.0 / 4.7 + 1.5, _n_.
  print 3.8 / 3.4 + 9.4, _n_.
  print 5.1 / 5.9 + 1.0, _n_.
  print 4.7 / 5.8 + 6.7, _n_.
  print 8.2 / 9.6 + 3.1, _n_.
  print 8.2 / 6.4 + 5.9, _n_.
  print 5.3 / 9.8 + 6.7, _n_.
  print 9.0 / 8.5 + 2.5, _n_.
  print 6.5 / 7.7 + 1.5, _n_.
  print 1.0 / 8.8 + 3.7, _n_.
  print 4.1 / 6.1 + 3.1, _n_.

  print 5.5 + 1.8 / 8.9 + 8.1, _n_.
  print 5.0 + 5.4 / 9.4 + 7.2, _n_.
  print 7.8 + 9.5 / 8.5 + 7.6, _n_.
  print 9.0 + 9.5 / 1.8 + 8.7, _n_.
  print 6.9 + 8.6 / 8.9 + 7.2, _n_.
  print 8.3 + 5.1 / 4.9 + 7.7, _n_.
  print 5.7 + 7.1 / 4.0 + 7.2, _n_.
  print 8.6 + 6.0 / 1.0 + 1.0, _n_.
  print 1.6 + 3.8 / 5.5 + 7.6, _n_.
  print 4.1 + 4.2 / 1.4 + 3.3, _n_.
  print 1.1 + 2.7 / 8.5 + 9.2, _n_.
  print 6.1 + 1.2 / 2.4 + 8.1, _n_.

  print 2.7 / 5.6 + 5.4 / 4.9, _n_.
  print 8.0 / 3.2 + 6.4 / 2.8, _n_.
  print 1.1 / 1.7 + 8.9 / 3.4, _n_.
  print 2.8 / 5.1 + 2.4 / 7.4, _n_.
  print 3.3 / 4.5 + 9.4 / 8.9, _n_.
  print 6.8 / 8.7 + 9.5 / 7.4, _n_.
  print 9.7 / 8.6 + 8.2 / 6.4, _n_.
  print 5.1 / 1.6 + 1.2 / 2.0, _n_.
  print 2.1 / 1.8 + 2.1 / 9.8, _n_.
  print 2.4 / 1.0 + 7.1 / 7.6, _n_.
  print 8.1 / 4.6 + 2.0 / 3.6, _n_.
  print 5.2 / 9.7 + 6.5 / 6.7, _n_.

}

