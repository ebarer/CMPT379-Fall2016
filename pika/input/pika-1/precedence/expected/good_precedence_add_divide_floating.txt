9.70625
4.32323
3.33505
10.5
5.49302
9.37027
5.9
9.35882
3.33288
8.17083
7.60588
9.01724
5.3875
2.98936
10.5176
1.86441
7.51034
3.95417
7.18125
7.24082
3.55882
2.34416
3.81364
3.77213
13.8022
12.7745
16.5176
22.9778
15.0663
17.0408
14.675
15.6
9.89091
10.4
10.6176
14.7
1.58418
4.78571
3.26471
0.873344
1.78951
2.06539
2.40916
3.7875
1.38095
3.33421
2.31643
1.50623
